frontend/src/features/admin-form/AdminFormCreatePage.stories.tsx

```diff
@@ -98,3 +120,14 @@ MobileAllFields.parameters = {
   chromatic: { viewports: [viewports.xs] },
   msw: buildMswRoutes({ form_fields: MOCK_FORM_FIELDS_WITH_MYINFO }),
 }
+export const MobileLoading = Template.bind({})
+MobileLoading.parameters = {
```

Comment:
nit: should we use the `getMobileViewParameters` storybook util function?

Reply:
sounds good!